cmd/build.go

```diff
@@ -64,40 +58,166 @@ var buildCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 
 		conf := getConfig()
+		fmt.Println(conf)
```

Comment:
Do we want to print the entire config?

Reply:
If we do, perhaps change this to only run in debug mode?

Reply:
I think removing it seems best, since there is currently no way to execute carpenter in debug or verbose mode.